lib/Doctrine/Connection/UnitOfWork.php

```diff
@@ -875,6 +891,8 @@ private function _updateCTIRecord(Doctrine_Table $table, Doctrine_Record $record
     /**
      * Class Table Inheritance code.
      * Support dropped for 0.10/1.0.
+     *
+     * @return array[]
```

Comment:
This is @return array[] but above in lib/Doctrine/Connection/Mssql.php there's a @return array

Reply:
This particular syntax means that it's an array of arrays, this is one that psalm probably auto-detected.

The type syntax of PHP docblocks supports marking what the array should contain, i.e.: `string[]` for an array of strings, or `Doctrine_Record[]` for an array of Doctrine_Record objects.  Some systems support more complicated syntax, allowing you to mark what type the keys are, etc...  but I think to have best compatibility, it's best to keep these simple.